### PR TITLE
Added support for single quotes

### PR DESCRIPTION
**Pull request by twolfson**
Hey @zheller, long time no see.

I am contributing to Suplemon which is using double quotes as its defaults. I would like to add quote linting to its test suite but currently `flake8-quotes` only supports linting for double quotes (i.e. their removal).

To resolve that, I have added a `quotes` option to the CLI/configuration files. In this PR:

- Added `quotes` option
- Replaced majority of mentioned for "double quotes" to "quotes"
- Fixed up quotes in README (irony, yey)
- Updated doubles tests to explicitly have "doubles_" in their filenames
- Added singles tests
- Added configuration documentation

**Notes:**

Upon landing this, please update the GitHub heading to be " Flake8 extension for checking quotes in python".

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -1,14 +1,29 @@
-Flake8 Extension to lint for double quotes.
+Flake8 Extension to lint for quotes.
 ===========================================
 
 Usage
 -----
 
-If you are using flake8 its as easy as:
+If you are using flake8 it's as easy as:
```
Comment (zheller) [inline README.rst]: lol good one

File: README.rst [hunk 2]
```diff
@@ -1,14 +1,29 @@
...
 
 .. code:: shell
 
     pip install flake8-quotes
 
-Now you don’t need to worry about people like @sectioneight constantly
+Now you don't need to worry about people like @sectioneight constantly
```
Comment (zheller) [inline README.rst]: ooo smart quote to dumb quote++

**Comment by zheller** [comment]
I appreciate the addition. Thanks!

**Comment by twolfson** [comment]
Awesome, thanks =) Any ETA on when the change will be published to PyPI?

**Comment by zheller** [comment]
Ill get it up tomorrow.

**Comment by twolfson** [comment]
Still waiting on a release for this =/

Btw, I don't mind becoming a collaborator to this project. The steps for that would be:

- Add as a collaborator on GitHub:
    - https://github.com/zheller/flake8-quotes/settings/collaboration
- Add as a maintainer (or owner) on PyPI:
    - https://pypi.python.org/pypi?:action=role_form&package_name=flake8-quotes

**Comment by zheller** [comment]
Done. Go crazy.

**Comment by twolfson** [comment]
Awesome, thanks @zheller =)

**Pull request merged**